app/models.py

```diff
@@ -0,0 +1,106 @@
+
+from sqlalchemy import Integer,BigInteger, Column, Boolean,String,Text,Date,ForeignKey,create_engine
+from sqlalchemy.orm import sessionmaker, relationship, declarative_base
+
+Base = declarative_base()
+
+class User(Base):
+    __tablename__ = 'users'
+
+    id = Column(Integer,primary_key=True,autoincrement=True)
```

Comment:
```suggestion
    id = Column(Integer,primary_key=True, autoincrement=True)
```
давай ставить после запятой пробел